Test/UnitTests/UnitTests.csproj

```diff
@@ -6,6 +6,8 @@
     <OutputType>Library</OutputType>
     <AssemblyName>UnitTests</AssemblyName>
     <RootNamespace>UnitTests</RootNamespace>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\UnitTests.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
```

Comment:
Do we even need this? IIRC, unsigned can reference strongnamed assemblies but not vice versa.

Reply:
Yes, it's needed because Microsoft.Xaml.Behaviors.dll has InternalsVisibleTo the UnitTests project which only works if the UnitTests assembly is strong-named afaik.

I mean, maybe the InmternalsVisibleTo can use `PublicKey=null`? But if not, then this is needed.

Reply:
Worth trying?